### PR TITLE
Witnesses cleaned up

### PR DESCRIPTION
**Pull request by polinavino**
And some other fixups

### CODE REVIEW

File: latex/ledger-spec.tex [hunk 1]
```diff
@@ -1451,10 +1483,9 @@ \subsection{Witnesses}
       \mathcal{V}_{\var{vk}}{\serialised{\txbody{tx}}}_{\sigma}
       \wedge  \fun{addr_h}~{utxoSt}~i = \hashKey{vk}\\
       ~ \\
-      & \forall (ix \mapsto (a, c)) \in outputs \cdot \exists! (\var{vk}, \sigma) \in \wits{\var{tx}}
+      & \forall c \in \fun{dcerts}~{tx} \cdot \exists! \dwit{c} = (\var{vk}, \sigma)
       \cdot \\
-      \mathcal{V}_{\var{vk}}{\serialised{(\txbody{tx}, (a,c))}}_{\sigma}
-      \wedge \fun{hashKey_{spend}} \var{a} = \hashKey{vk}\\
+      \verify{vk}{\serialised{(\dbody{c},~\txbody \var{tx})}}{\sigma}\\
```
[the remark this answers is not in the document]
Comment (polinavino) [reply latex/ledger-spec.tex]: This is a typo - it is supposed to be signing txdata, which has everything

[the remark this answers is not in the document]
Comment (JaredCorduan) [reply latex/ledger-spec.tex]: oh perfect! can we make `txdata` include witnesses as well?

[the remark this answers is not in the document]
Comment (polinavino) [reply latex/ledger-spec.tex]: we want to sign everything except witnesses i thought? and that is what that is meant to be

[the remark this answers is not in the document]
Comment (JaredCorduan) [reply latex/ledger-spec.tex]: sorry, yes, your plan is perfect! I was swapping witnesses and withdrawals in my head :)

**Pull request merged**